src/block_elements/button.rs

```diff
@@ -33,14 +33,18 @@ pub struct Contents {
     #[validate(length(max = 255))]
     action_id: String,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I wonder if we could write a macro to make this easy way to do concisely - like a macro at the struct level of `#[serde_skip_nones]`